flow/connectors/clickhouse/normalize.go

```diff
@@ -301,7 +301,18 @@ func (c *ClickHouseConnector) NormalizeRecords(
 					slog.Int64("syncBatchId", req.SyncBatchID),
 					slog.Int64("normalizeBatchId", normBatchID),
 					slog.String("query", query))
-				if err := chConn.Exec(errCtx, query); err != nil {
+
+				numParts, err := peerdbenv.PeerDBClickHouseNormalizationChunkingParts(ctx, req.Env)
+				if err != nil {
+					c.logger.Warn("failed to get chunking parts, proceeding without chunking", slog.Any("error", err))
+				} else if numParts > 1 {
```

Comment:
can we add "info" logging to indicate which branch we are in?

Reply:
the query will be logged & include the partition value at the end